src/redux/configStore.js

```diff
@@ -0,0 +1,16 @@
+import { configureStore } from '@reduxjs/toolkit';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import missionsReducer from './Missions/missions';
```

Comment:
- [ ] Kindly fix the file reference here too.

Reply:
Hi @phelian23 
Thanks for your feedback 
Can you please tell me which thing I should change in this file?

Reply:
@zakiyousufi The reference for the missionReducer should be `./missions/missions` with lowercase `m`

Reply:
@phelian23 
Thanks but after the changing I faced with these errors

![spaceship](https://user-images.githubusercontent.com/87229464/193427566-154a42af-d1e8-4d05-8916-6965852e87b1.JPG)

Reply:
How Can I solve these errors

Reply:
Can you get on a call so we look at it?

Reply:
@phelian23 
Yes, please check your slack or email I will send you an invitation link